docs/docs_config.js

```diff
@@ -0,0 +1 @@
+const versionNodes = []
```

Comment:
As we release new versions we can include them here (list of `{ version: <string> }`) and they will appear in the docs as a dropdown under the project name in the sidebar to allow for viewing of docs for different versions.